package.json

```diff
@@ -250,5 +254,16 @@
   "bugs": {
     "url": "https://github.com/Trustroots/trustroots/issues"
   },
-  "license": "MIT"
+  "license": "MIT",
+  "husky": {
+    "hooks": {
+      "pre-commit": "npm run -s install-dependencies && npx lint-staged"
+    }
+  },
+  "lint-staged": {
```

Comment:
An alternative for `lint-staged` could be [`pretty-quick](https://www.npmjs.com/package/pretty-quick) but its geared towards Prettier only. With lint-staged we can also run linter on files.

Reply:
Yes, seems good to be able to have it doing general linting too. Although we also have linting in `npm run lint:watch` _and_ in the webpack dev server... (for client files), linting everywhere... (not to mention my editor is doing it too).

Personally, I would not add the git hooks to my own local setup, I don't know why but they feel invasive, maybe I can learn to love them though. But not much unlinted makes it through the 3 lint stages above anyway.

Reply:
Hehe yeah. We should drop linting either here _or_ in webpack dev-server+lint:watch. I like the simplicity of this setup and since I have continuous linting in editor, I don't really need the dev-server linting. I appreciate it might be easier to approach for new devs/volunteers, tho.

What do you think?

In future I'd like to add Stylelint to lint *.less files so it would get added here, too.